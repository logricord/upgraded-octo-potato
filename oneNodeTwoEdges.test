# Test graph with one node and two edges

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 node1

# Add two edges
AddEdge graph1 node1 node1 edge1
AddEdge graph1 node1 node1 edge2

# Print the nodes in the graph and the outgoing edges from node1
ListNodes graph1
ListChildren graph1 node1