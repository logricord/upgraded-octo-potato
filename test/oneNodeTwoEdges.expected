# Test graph with one node and two edges

# Create a graph
created graph graph1

# Add a node
added node node1 to graph1

# Add two edges
added edge edge1 from node1 to node1 in graph1
added edge edge2 from node1 to node1 in graph1

# Print the nodes in the graph and the outgoing edges from node1
graph1 contains: node1
the children of node1 in graph1 are: node1(edge1) node1(edge2)